CourseApp/Program.cs

```diff
@@ -6,8 +6,33 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            Console.WriteLine($"Hello world");
+            double y = 0;
+            double x_n = 1.25;
```

Comment:
Please, do not name variables with `_`

Reply:
Can we add some rule for Style cop to avoid such cases

Reply:
@jskonst yes, https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/NamingRules.md